### DISCUSSION TITLE
加入安装mrpack（Modrinth整合包）的选项（急）

### DISCUSSION THREAD

**Discussion by hiteaCraft** [Meloong-Git/PCL#2211] [讨论]
加入安装mrpack（Modrinth整合包）的选项! 不想用bakaXL和HMCL awa

**Comment by Chiloven945** [comment]
那这是啥？
![image](https://github.com/Hex-Dragon/PCL2/assets/116699482/516ced12-7714-4088-b3c6-5a5009ce65c9)

**Comment by Pigeon0v0** [comment]
PCL2 在 2.5.0 就支持了

同时，请尽可能少的提及其他启动器，除非必要
